soe/src/stanford.make

```diff
@@ -152,7 +152,7 @@ projects[stanford_manage_content][download][url] = "git@github.com:SU-SWS/stanfo
 projects[stanford_manage_content][subdir] = "stanford"
 projects[stanford_manage_content][type] = "module"
 
-projects[stanford_news][download][tag] = "7.3.7-alpha5"
+projects[stanford_news][download][tag] = "7.x-3.7-alpha5"
```

Comment:
Tag name for semver was `7.3.7-alpha.5` but `7.x-3.7-alpha5` is also available

Reply:
https://github.com/SU-SWS/stanford_news/releases/tag/7.3.7-alpha.5
https://github.com/SU-SWS/stanford_news/releases/tag/7.x-3.7-alpha5